features/Roles.feature

```diff
@@ -122,7 +122,7 @@ Feature: Roles management
       And I click on the edit action bar button "Discard changes"
     Then I should be on "Role" "Test Role edited" page
       And "Policies" list in "Role" "Test Role edited" is empty
-      And There's an assignment "Subtree of Location: /Media/Images" for "Anonymous User" on the "Test Role edited" assignments list
+      And there is an assignment "Subtree of Location: /Media/Images" for "Anonymous User" on the "Test Role edited" assignments list
 
   @javascript @common
   Scenario: Policies can be added to role
```

Comment:
I'd change the name to something like "Limitations can be specified when creating a Role"

Reply:
I agree, but maybe more like "Policies with Limitations can be added to role", to be more specific?